src/sql/workbench/parts/notebook/browser/outputs/renderers.ts

```diff
@@ -478,10 +478,15 @@ namespace Private {
 		let anchors = node.getElementsByTagName('a');
 		for (let i = 0; i < anchors.length; i++) {
 			let path = anchors[i].href || '';
-			const isLocal =
-				resolver && resolver.isLocal
+			let isLocal: boolean;
```

Comment:
can we make a helper method here? I see the same logic in 3 places 😄

Reply:
fixed